Translate/src/V2/TranslateClient.php

```diff
@@ -113,7 +115,7 @@ class TranslateClient
      *     @type string $quotaProject Specifies a user project to bill for
      *           access charges associated with the request.
      * }
-     * @throws \InvalidArgumentException
+     * @throws \InvalidArgumentException|GoogleException
```

Comment:
Digging into this, I do not think that actually throws `\InvalidArgumentException`, and that can be removed.

Also, if we have more than one `@throws`, it should be chained with an additional `@throws` PHP doc instead of using the `|` operator, for readability.